### ISSUE TITLE
make a new place for pymacs

### ISSUE THREAD

**Issue by glyph** [pinard/Pymacs#65]
Given that François has passed away, and that a new maintainer has stepped up, it seems that this should no longer be the official repository for Pymacs in any way.

I suggest making a new github org, moving @dgentry's fork there, and setting up an issue tracker there.

**Comment by dgentry** [comment]
This is fine with me too.

**Comment by Awannaphasch2016** [comment]
any progress?

**Comment by QiangF** [comment]
Looks like no body is in charge. Why not make a new package? For example pymacs2

**Comment by dgentry** [comment]
“I suggest making a new github org, moving @dgentry's fork there, and setting up an issue tracker there.”

I wasn’t volunteering to do it personally, but ok, done. There’s https://github.com/org/pymacs2 org now, with 1 repo, which is a fork of dgentry/Pymacs.   I enabled an Issue tracker and Discussions. I like QiangF’s idea of a pymacs2 package for pypi, but again, not volunteering to do it. (I might eventually look into it, but no commitment.)

The org name couldn’t be pymacs, which was already taken by an apparently lifeless GitHub account, so pymacs2 it is.

Do you want to join as a contributor?

**Comment by QiangF** [comment]
Yes, please add me. I think that many people in the emacs-eaf community will be interested in this project. https://github.com/emacs-eaf/emacs-application-framework

**Comment by QiangF** [comment]
So the home to the new package is https://github.com/pymacs2
